web-ui/controller/git_forge.ml

```diff
@@ -1,8 +1,9 @@
+open Lwt.Infix
```

Comment:
Check your usages of this module, some places you re-open it for a single function.
Ideally I would keep the scope this is open in as short as possible.